plugin/src/test/java/io/jenkins/plugins/analysis/core/steps/MatrixBridgeTest.java

```diff
@@ -0,0 +1,68 @@
+package io.jenkins.plugins.analysis.core.steps;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import hudson.Launcher;
+import hudson.matrix.MatrixAggregator;
+import hudson.matrix.MatrixBuild;
+import hudson.matrix.MatrixProject;
+import hudson.model.BuildListener;
+import hudson.util.DescribableList;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+/**
+ * Tests the class {@link MatrixBridge}.
+ *
+ * @author Naveen Sundar
+ */
+public class MatrixBridgeTest {
+    // to be tested
+    private MatrixBridge matrixBridge;
+
+    // Dependencies
+    private MatrixBuild build;
```

Comment:
Fields are not allowed in tests. Can you please refactor and inline in a method that is called by both tests? You may need to inline `TaskListener.NULL` and launcher as well.

Reply:
ok